docs/reference/ccr/getting-started.asciidoc

```diff
@@ -1,7 +1,320 @@
 [role="xpack"]
 [testenv="platinum"]
-[[ccr-getting-started]]
-== Getting Started
+[[ccr-gs]]
+== Getting Started with Cross-Cluster Replication
```

Comment:
It would be better for SEO purposes to leave "getting-started" spelled out in this anchor. The subsections can use the abbreviation.  Should probably also use the {ccr} attribute in the header.